react/src/components/molecules/ArrowNav/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {ArrowButton} from '../../../index';
+import './style.css';
+
+const ArrowNav = (props) => {
+  const sectionClasses = classNames({
+    'ma__arrow-nav': true,
+    [`ma__arrow-nav--${props.direction}`] : props.direction
+  });
+  const Element = props.href ? 'a' : 'button';
+  const onClickCallback = (e) => {
+    if (typeof props.onClick === 'function') {
+      e.preventDefault();
+      props.onClick(e);
+    }
+  };
+  return (
+    <section className={sectionClasses}>
```

Comment:
I think it might be better to render this using this.props.children, so that you don't have to set all of ArrowButton's props using ArrowNav's props.

Reply:
So since the whole section is clickable, the only thing I want to pass to the ArrowButton is the direction. All actions are handled now. See update